docs/api-reference/orthographic-viewport.md

```diff
@@ -26,10 +26,14 @@ Parameters:
   * `left` (Number) - Left bound of the frustum
   * `top` (Number) - Top bound of the frustum
 
-  automatically calculated:
+  automatically calculated by default:
```

Comment:
Given this is a major release, we can make this api cleaner and document it in `upgrade-guide`.
1. Make `right` and `bottom` non optional.  Specifying just `top` and `left` and calculating `right` and `bottom` looks asymmetric any way.
2. Replace `top`, `left`, `bottom` and `right`, with `scale`, inside `OrthographicViewport` we can calculate all 4 parameters using `width`, `height` and `scale`.

Reply:
I prefer to keep these parameters, it allows the users to specify the range, [-1, 1] or [0, 1] per use case. Using scale instead assumes [-1, 1] which is not always the case.